### PR TITLE
Browser extension keyboard shortcuts

### LINKED ISSUE #14: Add keyboard shortcuts for calendar navigation: Browser Extension

### PR DESCRIPTION
**Pull request by paudsu01**
Add keyboard shortcuts to open/close and navigate around the patro chrome extension. 

A background.js service worker has been added which listens for keyboard shortcut(command) inputs as for our extension and sends message to the script.js for the popup to execute different functions.

It is important to note that the shortcuts are enabled only when the popup is open except for the command to open the extension itself.

### CODE REVIEW

**Comment by paudsu01** [comment]
Chrome only allows to suggest 4 keyboard shortcuts by default. This pull request has a total of 7 added commands, the remaining 3 won't be setup initially and can be setup by the user from `chrome://extensions/shortcuts`. Here is what the commands look like: 
image
Here are my recommended shortcuts : 
image

**Comment by paudsu01** [comment]
@sumanchapai, please review the changes made. Let me know if everything is good, after which I will update the extension on the webstore.

File: browser_extension/script/script.js [hunk 1]
```diff
@@ -241,6 +281,118 @@ function displayDayInformation(TdElement){
     highlightCurrentDay(false);
 }
 
+function show_today(){
```
Comment (sumanchapai) [inline browser_extension/script/script.js]: camelCase

File: browser_extension/script/script.js [hunk 2]
```diff
@@ -241,6 +281,118 @@ function displayDayInformation(TdElement){
...
+    displayDayInformation(document.getElementById(elementID));
+    }
+}
+
+function show_previous_day(){
+
+    let selected_date_array = currentSelectedTdElementID.split('-');
+    let year = selected_date_array[0];
+    let monthIndex = selected_date_array[1] - 1;
+    let day = selected_date_array[2];
```
Comment (sumanchapai) [inline browser_extension/script/script.js:351]: camelCase

File: browser_extension/script/script.js [hunk 3]
```diff
@@ -241,6 +281,118 @@ function displayDayInformation(TdElement){
...
+    } else {
+    // But we need to change calendar month if needed
+    changeMonthAndYear('next');
+    displayDayInformation(document.getElementById(elementID));
+    }
+}
+
+function show_previous_day(){
+
+    let selected_date_array = currentSelectedTdElementID.split('-');
```
Comment (sumanchapai) [inline browser_extension/script/script.js:348]: camelCase

**Review by sumanchapai — COMMENTED** [review]
try to use camelCase in JS files

**Comment by paudsu01** [comment]
@sumanchapai, please check the latest changes. Do approve and merge the pull request if you don't see any issues. Thanks for the feedback.

**Review by sumanchapai — APPROVED** [review]

**Pull request merged**